grunt/config/rollup.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+'use strict';
+
+var grunt = require('grunt');
+var rollupBabel = require('rollup-plugin-babel');
+var rollupReplace = require('rollup-plugin-replace');
+var rollupResolve = require('rollup-plugin-node-resolve');
+var UglifyJS = require('uglify-js');
+
+var babelEs6ModulifyTransform = require('../../scripts/babel/transform-es6-modulify');
+
+var SIMPLE_TEMPLATE =
+  grunt.file.read('./grunt/data/header-template-short.txt');
+
+var LICENSE_TEMPLATE =
+  grunt.file.read('./grunt/data/header-template-extended.txt');
+
+function minify(src) {
+  return UglifyJS.minify(src, {fromString: true}).code;
+}
+
+function bannerify(src) {
+  var version = grunt.config.data.pkg.version;
+  var packageName = this.data.packageName || this.data.moduleName;
+  return (
+    grunt.template.process(
+      LICENSE_TEMPLATE,
+      {data: {package: packageName, version: version}}
+    ) +
+    src
+  );
+}
+
+function simpleBannerify(src) {
```

Comment:
Let's stop using this. The only reason we did it before was because we didn't lose the license header at the top of every file when bundling. But we do lose those now with rollup so we should have the full license header.